velox/benchmarks/tpch/TpchBenchmark.cpp

```diff
@@ -127,15 +136,31 @@ BENCHMARK(q18) {
 
 int main(int argc, char** argv) {
   folly::init(&argc, &argv, false);
+  bool runQueryVerboseSet = isFlagSet("run_query_verbose");
```

Comment:
Why can't you use FLAGS_run_query_verbose variable?

Reply:
`FLAGS_run_query_verbose` is a variable of the flag's value type (int32) here. There is no way to get the flagname since we use pragmas to define them `DEFINE_int32(run_query_verbose ...`

Reply:
Not sure I understand. Why do you need the flag name? Can't you just check if (FLAGS_run_query_verbose > 0) ?

Reply:
We want to know here if the flag is specified as an argument or not. The only way I see is to use the logic inside `isFlagSet`(gflags tells us if the default is being used. It does not look at the actual value). The current approach of manually checking against the default values is not always clean. For example: `include_custom_stats` is a bool. If we check against the default value `false` to see if it is specified, it is not always right. The user might be using some tooling that specifies "--include_custom_flag=false". Here the flag is specified but we will infer that the flag is not specified since we are looking a the value `bool`.

Reply:
This is very confusing. Why can't we write the check as follows?

```
if (!FLAGS_run_query_verbose && FLAGS_include_custom_stats) {
   ....throw not supported
}
```

Reply:
Also, why can't we simply ignore FLAGS_include_custom_stats if FLAGS_run_query_verbose is false and document that include_custom_stats flag applies only if include_custom_stats is on?